### DISCUSSION TITLE
Export to Latex

### DISCUSSION THREAD

**Discussion by IpadDilectio** [nvim-neorg/neorg#939] [Ideas]
Is it possible to implement direct latex export?

At the moment, if I write a $ Definition and export it in Markdown, it doesn't appear correctly in the export. In fact, the name of the definition appears at the end of its description. On the other hand, if the export were done directly in latex, with the command latex \newtheorem it would be possible to retain the useful structure of neorg while making input easier than directly in latex.

**Comment by pysan3** [comment]
The pandoc parsers might help you out. 

Use one of the followings (I think norganic works better imc) to output json and pipe it into pandoc to generate latex or possibly a pdf directly. (Norg-pandoc works in a plugin style)

- https://github.com/klafyvel/norganic
- https://github.com/boltlessengineer/norg-pandoc

**Reply by IpadDilectio** [reply]
Hello,
Thank you for your reply. I had indeed been thinking about Pandoc, but I didn't know about the two modules you mentioned. I'd rather try the second one, but I've already exported my Markdown notes with Norg, so it's quite easy to use Pandoc afterwards to transfer them to LaTeX.
Just bear in mind that not all LaTeX functions can be used in Norg, and consider adding them directly to the LaTeX document.
I only use Norg now to collect drafts of articles I want to write later. Once that's understood, Norg remains a very useful tool.
If certain LaTeX functions could be taken into account directly in Norg, that would be great, but I can imagine that few people are likely to be interested in this kind of possibility.
Thanks again.
Sincerely

**Reply by pysan3** [reply]
Well yah, I guess many of the functionalities are not complete or have bugs here and there.

A good thing tho is that we are now working on a new version of the parser so it should be able to comprehend norg file better and a pandoc parser will be implemented based on that.

Or you could wrap raw latex code with `ranged tags` and write your own macro for that tag to export it to latex directly.
Sad thing is, that this feature is not implemented yet lol.
